settings.gradle.kts

```diff
@@ -1,4 +1 @@
-
 rootProject.name = "ktor-io"
-
-include(":core")
```

Comment:
Not sure, if it's a good change, as I believe 1 module will be not enough for `io` library :)

Reply:
Let's not introduce complexity in advance, will add later when it will be necessary